lib/diagnostic.js

```diff
@@ -4,54 +4,70 @@
 //
 // NOTE: Line numbers are annotated line numbers, not editor line numbers.
 
-/*1*/ let alderaan = 'Boom';
-/*2*/ let deathStar = function deathStar(){
-/*3*/   let laserQuantity = 1;
-/*4*/   let laserSize = 'huge';
-/*5*/   let fire = function fire(){
-/*6*/     // some code
-/*7*/   };
-/*8*/ };
+/*1*/
+let alderaan = 'Boom';
+/*2*/
+let deathStar = function deathStar() {
+  /*3*/
+  let laserQuantity = 1;
+  /*4*/
+  let laserSize = 'huge';
+  /*5*/
+  let fire = function fire() {
+    /*6*/ // some code
+    /*7*/
+  };
+  /*8*/
+};
 
 // Suppose we had declared another function between line 7 and 8. Would that
 // new function have access to the value stored in the variable `laserQuantity`
 // on line 3?
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = true; // replace null with your response
 //
 
 // Which scope(s) may access the variable `laserQuantity`? Choose one or many
 // of the following:
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-let scopes = [/* your response here */];
+let scopes = [ /* your response here */ ];
```

Comment:
we didn't explicitly cover this in class, but the answer here should be `['deathStar', 'fire']`

`laserQuality` is a parameter inside `deathStar`, so `deathStar` would have access to it. `fire` is declared in the same scope as `laserQuality` and therefore would also access to it.